### PR TITLE
initial commit, project structure

### PR DESCRIPTION
**Pull request by ViktoriyaSofia**
ok

### CODE REVIEW

**Comment by ViktoriyaSofia** [comment]
ok

**Comment by MaksymBilyk4** [comment]
accept my pull request!!!!!!!!!!!!!!!!!!!!!1

**Comment by Vladry** [comment]
not allowed!

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,9 @@
 # Log file
 *.log
 
+# Idea
+.idea
```
Comment (Vladry) [inline .gitignore:8]: поставь в конце косую    .idea/

**Pull request closed without merging**